hw/ip/usbdev/doc/_index.md

```diff
@@ -251,8 +218,15 @@ In addition, the buffer ID, the packet size, an out/setup flag and the endpoint
 Software should immediately provide a free buffer for future reception by writing the corresponding buffer ID to the Available Buffer FIFO.
 It can then process the packet and eventually return the received buffer to the free pool.
 This allows streaming on a single endpoint or across a number of endpoints.
-If the packets cannot be consumed at the rate they are received, software can implement selective flow control by disabling OUT or SETUP transactions for a particular endpoint, which will result in a request to that endpoint being NACKed (negative acknowledgment).
-In the unfortunate event that the Available Buffer FIFO is empty or the Received Buffer FIFO is full, all OUT and SETUP transactions are NACKed.
+If the packets cannot be consumed at the rate they are received, software can implement selective flow control by disabling OUT transactions for a particular endpoint, which will result in a request to that endpoint being NAKed (negative acknowledgment).
```

Comment:
this is different from the end point being completely disabled right? I recall there was a discussion on an interface that's disabled should not even respond.

Reply:
That's correct. In truth, I probably should rename the `rxenable_out` register somehow... "Enable" / "Disable" isn't really the right verb... It's more like "Accept" / "Reject" (except "Stall" overrides, from a separate register).

Do you have a preference?

Reply:
ah there's no need to change, i was really just asking a clarifying question.